polling_stations/templates/base_full.html

```diff
@@ -0,0 +1,42 @@
+{% load i18n %}
```

Comment:
I don't think we need this file, as we will create the embed file with blocks instead